### DISCUSSION TITLE
API key

### DISCUSSION THREAD

**Discussion by russrimm** [iopool/community#21] [General]
Hi all,
Just got my iopool device and found out it has an API key which might be fun.  I was able to use it to hit the /pool endpoint, but when I tried the /pool/:id it says "Missing authentication token".  Maybe i'm not understanding the docs, but I tried /pool/:id, /pool/1234567 (but using my actual pool ID from the returned id value from /pool), /pool/:1234567.  What am I doing wrong?  
Also is it possible for the API to return the temp in Fahrenheit or do I have to do the conversion on my own from the response?
Thanks

**Comment by pixelmixer** [comment]
The url is part of the solution, but importantly you need to also include the actual API Key from the app in the "x-api-key" header in your request. Otherwise it'll never work. 

https://help.iopool.com/en/articles/5537423-iopool-public-api
